src/bsoncxx/document/view.hpp

```diff
@@ -121,6 +121,13 @@ class BSONCXX_API view {
     ///
     std::size_t length() const;
 
+    ///
+    /// Checks if the underlying document is empty
```

Comment:
I would elaborate a bit: 

```Checks if the underlying document is empty, i.e. it is equivalent to the trivial document '{}'.```

Also please put periods at the end of sentences.